src/picongpu/include/particles/traits/GetIonizer.hpp

```diff
@@ -34,6 +34,8 @@
 namespace picongpu
 {
 
+namespace traits
```

Comment:
all other traits in this folder are in `picongpu::traits` therefore I only change the different to keep the over all change small